frontend/app/routes/_public+/apply+/$id+/terms-and-conditions.tsx

```diff
@@ -0,0 +1,173 @@
+import { ActionFunctionArgs, LoaderFunctionArgs, json, redirect } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+
+import { useTranslation } from 'react-i18next';
+
+import { ButtonLink } from '~/components/buttons';
+import { CollapsibleDetails } from '~/components/collapsible';
+import { InlineLink } from '~/components/inline-link';
+import { getApplyFlow } from '~/routes-flow/apply-flow';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { RouteHandleData } from '~/utils/route-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('terms-and-conditions');
+
+export const handle = {
+  breadcrumbs: [
+    {
+      labelI18nKey: 'terms-and-conditions:terms-and-conditions.index.breadcrumbs.terms-and-conditions',
+    },
+  ],
```

Comment:
Suggestion: The breadcrumbs for all pages of the CDCP application appear to be the same in the Figma mock-ups:
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/eed3671e-41b6-443a-a839-d39d42efc676)


Consider moving the breadcrumbs into the layout instead (in a future PR?).

Reply:
Removed breadcrumbs for now.